celery/worker/consumer/delayed_delivery.py

```diff
@@ -0,0 +1,82 @@
+from kombu import Exchange, Queue
+
+from celery import Celery, bootsteps
+from celery.utils.log import get_logger
+from celery.worker.consumer import Consumer, Tasks
+
+__all__ = ('DelayedDelivery',)
+
+logger = get_logger(__name__)
+
+
+class DelayedDelivery(bootsteps.StartStopStep):
```

Comment:
Please add documentation to this class indicating how it implements the interface etc.